APMrover2/AP_MotorsUGV.h

```diff
@@ -138,5 +156,13 @@ class AP_MotorsUGV {
     float   _throttle;  // requested throttle as a value from -100 to 100
     float   _throttle_prev; // throttle input from previous iteration
     bool    _scale_steering = true; // true if we should scale steering by speed or angle
-    float   _lateral;  // requested lateral input as a value from -4500 to +4500
+    float   _lateral;  // requested lateral input as a value from -100 to +100
+
+    // custom configs' variables
+    bool    _motor_enabled[UGV_MAX_NUM_MOTORS];
+    float   _throttle_factor[UGV_MAX_NUM_MOTORS];
+    float   _steering_factor[UGV_MAX_NUM_MOTORS];
+    float   _lateral_factor[UGV_MAX_NUM_MOTORS];
+    float   _output_vectored;
+    float   _config_num_of_motors;
```

Comment:
the main purpose of _config_num_of_motors is to avoid errors in pre_arm_check(), otherwise it's not that necessary. I reckon _motor_enabled can also be removed, I'll remove it and do a quick test.